.github/workflows/build.yml

```diff
@@ -60,6 +60,11 @@ jobs:
           check-latest: true
           cache: true
 
+      - name: Set up Clang
```

Comment:
Should we mention here and below that this is installing not just clang but all the other related LLVM project tools?